src/map.js

```diff
@@ -16,21 +16,30 @@ const DEFAULT_OPTIONS = {
         color: '#0CB1E8',
         weight: 1,
         opacity: 0.5,
-        smoothFactor: 1
+        smoothFactor: 1,
+        existing: true,
```

Comment:
Could we give this a more descriptive name? By itself, `existing` doesn't really explain much.

Reply:
Fixed in cbe07f92b6c863c844d099c9531816bb4330c897